src/board/ui.ts

```diff
@@ -18,12 +19,34 @@ export function createBoard(
 ) {
   document.body.insertAdjacentHTML("afterbegin", svgText);
   const svg = document.querySelector("svg");
-  // We start stopped.
   if (!svg) {
     throw new Error("No SVG");
   }
-  svg.style.opacity = stoppedOpactity;
-  return new BoardUI(operations, fs, svg, onSensorChange);
+  const stoppedOverlay = createStoppedOverlay();
+  svg.insertAdjacentElement("afterend", stoppedOverlay);
+  return new BoardUI(operations, fs, svg, stoppedOverlay, onSensorChange);
+}
+
+function createStoppedOverlay() {
+  const stoppedOverlay = document.createElement("div");
+  stoppedOverlay.classList.add("play-button-container");
+  const playButton = document.createElement("button");
+  playButton.classList.add("play-button");
+  const color = getButtonColor();
+  if (color) {
+    playButton.style.color = color;
+    playButton.style.borderColor = color;
+  }
+  playButton.insertAdjacentHTML("beforeend", playIcon);
+  stoppedOverlay.appendChild(playButton);
+  return stoppedOverlay;
+}
+
+function getButtonColor() {
+  const params = new URLSearchParams(window.location.search);
+  if (params.has("color")) {
+    return params.get("color");
+  }
 }
```

Comment:
Do we need the `has` check? It returns null if missing which might be find for our purposes.

Even if intentional, returning undefined like this by falling off the end of the function looks like a bug. If we want that behaviour then let's do it explicitly and type the return value.

Reply:
[`c01dc04`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/c01dc04b348978a971f469467a68a8d3394dc0b3).